xbmc/filesystem/FileCache.h

```diff
@@ -61,6 +61,12 @@ namespace XFILE
 
     const std::string GetProperty(XFILE::FileProperty type, const std::string &name = "") const override;
 
+    virtual const std::vector<std::string> GetPropertyValues(XFILE::FileProperty type, const std::string &name = "") const
+    {
+      std::vector<std::string> values;
+      return values;
```

Comment:
No need for a variable here. `return std::vector<std::string>();` is sufficient.